### ISSUE TITLE
Crashes When Typing in Search Bar

### ISSUE THREAD

**Issue by dark-gamer456** [Cy4Shot/moai#3]
I have the models in the models/moai/tabula folder and when i try to search the model name in the moai block, I type the first letter and then it crashes

Crash report:
[crash-2021-06-12_10.33.15-client.txt](https://github.com/Cy4Shot/moai/files/6642634/crash-2021-06-12_10.33.15-client.txt)

**Comment by Cy4Shot** [comment]
seems like its a problem with the model itself, not the search bar. can you send me the model?

**Comment by dark-gamer456** [comment]
I dont think its the model because when type the first letter of it being a 'T' it immediatly crashes, not the whole model name

**Comment by Cy4Shot** [comment]
> I dont think its the model because when type the first letter of it being a 'T' it immediatly crashes, not the whole model name

nope. it searches for the model straight away while you type and tries to render it
